src/styles/header.scss

```diff
@@ -0,0 +1,84 @@
+:root {
+  --active-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  padding: 0 50px;
+  box-sizing: border-box;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.logo_pic {
```

Comment:
Class names should be written in kebab-case, not snake_case for consistency and readability. Rename 'logo_pic' to 'logo-pic'.